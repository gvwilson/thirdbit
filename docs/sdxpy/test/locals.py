def show_locals(low, high):
    print(f"start: {locals()}")
    for i in range(low, high):
        print(f"loop {i}: {locals()}")
    print(f"end: {locals()}")

show_locals(1, 3)
