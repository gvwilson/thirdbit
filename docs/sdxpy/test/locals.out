start: {'low': 1, 'high': 3}
loop 1: {'low': 1, 'high': 3, 'i': 1}
loop 2: {'low': 1, 'high': 3, 'i': 2}
end: {'low': 1, 'high': 3, 'i': 2}
